planpro_importer/reader.py

```diff
@@ -46,28 +46,29 @@ def read_topology_from_container(self, container):
 
         for top_kante in container.TOP_Kante:
             top_kante_uuid = top_kante.Identitaet.Wert
+            length = top_kante.TOP_Kante_Allg.TOP_Laenge.Wert
             node_a = self.topology.nodes[top_kante.ID_TOP_Knoten_A.Wert]
             node_b = self.topology.nodes[top_kante.ID_TOP_Knoten_B.Wert]
+            edge = Edge(node_a, node_b, length=length, uuid=top_kante_uuid)
```

Comment:
Are you sure that ```top_kante_uuid``` is in scope here? It is defined and assigned in a separate for loop.

Reply:
It's defined in line 48 🤔

Reply:
I don't know how I missed that